scripts/enterGame.js

```diff
@@ -13,35 +13,33 @@ function start() {
 
 // Changes the title on the UI
 function changeTitle() {
-    title = "Rock Paper Scissors";
+    titleElement = "Rock Paper Scissors";
 }
 
 // Removes the start button from the UI
 function removeUI() {
-    start = "";
+    startElement = "";
 }
 
 // Draws the entire game, by writing in all the new codes.
-function drawUI {
+function drawUI() {
     // Define draw and drawElement
-    draw = "";
     drawElement.innerHTML = "";
-    // Run the 3 functions to setup the codes.
-    drawNameBox();
-    drawSelection();
-    drawDisplay();
     // Add the codes to be displayed on the user interfaces.
-    drawElement.appendChild(drawNameBox);
-    drawElement.appendChild(drawSelection);
-    drawElement.appendChild(drawDisplay);
+    drawElement.appendChild(drawNameBox());
```

Comment:
You are trying to append `drawNameBox` which is a function. I moved the login in here and made the functions return the HTML element